### PR TITLE
JIRA: ABC-96 Resetting AlembicStreamPlayer settings removes Stream Descriptor

### PR DESCRIPTION
**Pull request by clusty**
Create a menu disabled entry that "hides" the Inspector reset one.

### CODE REVIEW

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a bug that cased data loss when the AlembicStreamPlayer was reset.
```
Comment (judubu) [inline com.unity.formats.alembic/CHANGELOG.md:11]: Typo. Should be "caused"

File: com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs [hunk 2]
```diff
@@ -295,5 +295,16 @@ void UndoRedoPerformed()
 
             loadSucceded = (target as AlembicStreamPlayer).ReloadStream();
         }
+
+        [MenuItem("CONTEXT/AlembicStreamPlayer/Reset"), ]
+        static void ResetPreventer()
```
Comment (judubu) [inline com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs:300]: Not sure I'm reading this right. It's a context menu that's always disabled and does nothing?

**Review by judubu — CHANGES_REQUESTED** [review]

**Comment by clusty** [comment]
Basically i am disabling the reset button.

File: com.unity.formats.alembic/CHANGELOG.md (hunk 1 above)
Comment (sebastienduverne) [reply com.unity.formats.alembic/CHANGELOG.md:11]: suggestion:
`- Fixed a bug that was causing data loss when resetting the AlembicStreamPlayer.`

File: com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs (hunk 2 above)
Comment (judubu) [reply com.unity.formats.alembic/Editor/Importer/AlembicStreamPlayerEditor.cs:300]: All good.

**Review by judubu — APPROVED** [review]

**Review by sebastienduverne — APPROVED** [review]
Changelog OK.

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2021.2.0a16 (Windows only)

**Pull request merged**